binderhub/app.py

```diff
@@ -19,7 +19,7 @@
 from .launcher import Launcher
 from .registry import DockerRegistry
 from .main import MainHandler, ParameterizedMainHandler, LegacyRedirectHandler
-from .repoproviders import GitHubRepoProvider, GitRepoProvider, GitLabRepoProvider
+from .repoproviders import GitHubRepoProvider, GitRepoProvider, GitLabRepoProvider, GitHubGistRepoProvider
```

Comment:
nitpick: can this be `GitHubGistProvider` or even `GistProvider`? I don't know anywhere except GitHub that has Gists.